keadm/cmd/keadm/app/cmd/debug/check.go

```diff
@@ -18,6 +18,8 @@ import (
 
 	"github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/common"
 	"github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/util"
+
+	"github.com/kubeedge/kubeedge/common/constants"
```

Comment:
this should be moved to line21

Reply:
thanks, i install the 'goimports plugin' , and let it format for me when 'ctrl+s'.
Had updated.